firstvoices/backend/serializers/import_job_serializers.py

```diff
@@ -0,0 +1,136 @@
+import tablib
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+from tablib import InvalidDimensions
+
+from backend.models.import_jobs import ImportJob, ImportJobReport, ImportJobReportRow
+from backend.models.media import File
+from backend.permissions.predicates.base import is_superadmin
+from backend.serializers.base_serializers import (
+    CreateSiteContentSerializerMixin,
+    SiteContentLinkedTitleSerializer,
+)
+from backend.serializers.media_serializers import FileUploadSerializer
+from backend.serializers.utils import (
+    check_required_headers,
+    get_site_from_context,
+    validate_headers,
+)
+from backend.serializers.validators import SupportedFileType
+
+
+class ImportReportRowSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = ImportJobReportRow
+        fields = ["row_number", "status", "errors"]
+
+    row_number = serializers.IntegerField(read_only=True)
+    status = serializers.CharField(read_only=True)
+    errors = serializers.ListField(child=serializers.CharField(), read_only=True)
+
+
+class ImportReportSerializer(serializers.ModelSerializer):
+    total_rows = serializers.IntegerField(read_only=True)
+    diff_headers = serializers.ListField(child=serializers.CharField(), read_only=True)
+    rows = ImportReportRowSerializer(many=True, read_only=True)
+    totals = serializers.JSONField(read_only=True)
+
+    class Meta:
+        model = ImportJobReport
+        fields = ["total_rows", "diff_headers", "totals"]
+
+
+class ImportJobSerializer(
```

Comment:
Should this serialize also display the inherited `BaseJob` fileds, such as `status` and `task_id` if so it could inherit from the `BaseJobSerializer` in `job_serializers.py`

Reply:
This is a good idea. +1 from me